File: bookActivities/Activity10C3.c
#include <stdio.h>
#include <stdbool.h>
#include <string.h>
#include <time.h>
int main(){
	struct StudentRecord {
		char name[20];
		char classId[2];
		char dateOfBirth[10];
		bool gender;
	};
	
	struct StudentRecord myStudent;
	strcpy(myStudent.name,  "Joe Biden");
	strcpy(myStudent.classId, "5A");
	strcpy(myStudent.dateOfBirth, "2010/03/21");
	myStudent.gender = true;

}

File: bookActivities/Activity10C3.py
import datetime

class StudentRecord():
    def __init__(self):
        self.name = None
        self.classId = None
        self.dateOfBirth = None
        self.gender = None


myStudent = StudentRecord()

myStudent.name = "Ahmad Sayed"
myStudent.classId = "5A"
myStudent.dateOfBirth = datetime.datetime(2010, 3, 21)
myStudent.gender = True

